fission-rbac.yaml

```diff
@@ -0,0 +1,175 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fission-svc
+  namespace: fission
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1alpha1
```

Comment:
in kubernetes v1.6.2 it should be `apiVersion: rbac.authorization.k8s.io/v1beta1`